osmaxx-py/excerptconverter/baseexcerptconverter/base_excerpt_converter.py

```diff
@@ -0,0 +1,89 @@
+import abc as abstract_base_class
+import stored_messages
+
+from celery import shared_task
+
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+
+
+class BaseExcerptConverter(metaclass=abstract_base_class.ABCMeta):
+    available_converters = []
+    name = 'base'
+    export_formats = {}
+    export_options = {}
+    steps_total = 0
+
+    @classmethod
+    def converter_configuration(cls):
+        return {
+            'name': cls.name,
+            'formats': cls.export_formats,
+            'options': cls.export_options
+        }
+
+    @staticmethod
+    @abstract_base_class.abstractmethod
+    @shared_task
+    def execute_task(extraction_order_id, supported_export_formats, execution_configuration):
+        return None
+
+    @staticmethod
+    def inform_user(user, message_type, message_text, email=True):
```

Comment:
Would it make sense to move this function to the user class and make it an instance method?